modules/gdprEnforcement.js

```diff
@@ -198,9 +199,14 @@ function gdprRule(purposeNo, checkConsent, blocked = null, gvlidFallback = () =>
   return function (params) {
     const consentData = gdprDataHandler.getConsentData();
     const modName = params[ACTIVITY_PARAM_COMPONENT_NAME];
+
     if (shouldEnforce(consentData, purposeNo, modName)) {
       const gvlid = getGvlid(params[ACTIVITY_PARAM_COMPONENT_TYPE], modName, gvlidFallback(params));
-      let allow = !!checkConsent(consentData, modName, gvlid);
+      let allow =
```

Comment:
@dgirardi had one question about the logic here.  know we want to go with publisher.consent in this scenario.. but was wondering if i put the FIRST_PARTY_GVLID ternary in the right place?